sdk/js/src/api/stream/stream-node.js

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import axios from 'axios'
+import Token from '../../util/token'
+import { notify, EVENTS } from './shared'
+
+/**
+ * Opens a new stream.
+ *
+ * @param {Object} payload  - The body of the initial request.
+ * @param {string} url - The stream endpoint, defaults to `/api/v3/events`.
+ * @param {string} method  - The HTTP method for the initial request, defaults to `POST`.
```

Comment:
GET is not supported by the backend, so maybe just remove this to avoid confusion (also below)